datapacks/Scaffolding Rush/data/loumardes/functions/scaffolding_rush/broadcast/maxbuild.mcfunction

```diff
@@ -0,0 +1 @@
+tellraw @p ["",{"text":"[SR]","color":"gold"},{"text":" Tu n'as pas le droit de poser des blocks a cette hauteur !","color":"dark_red"}]
```

Comment:
english please

Reply:
tellraw @p ["",{"text":"[SR]","color":"gold"},{"text":" You can't place block here becase it is to high !","color":"dark_red"}] 

(fait avec mon super accent ; )